send_message.py

```
@@ -0,0 +1,208 @@
+import hl7apy
+from hl7apy.parser import parse_message
+import schedule
+
+
+TIME = datetime.datetime.now().timestamp()
+
+logger = logging.getLogger(__name__)
+
+
+def get_relevant_files(
+    folder: PosixPath, test: bool
```

Ruff (0.14.1)

Unused function argument: `test`

(ARG001)